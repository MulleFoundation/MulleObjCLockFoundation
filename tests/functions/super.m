#import <MulleObjC/MulleObjC.h>

#include "class-system.inc"


int   main( int argc, char *argv[])
{
   [A foo];
   printf( "\n");
   [B foo];
   return( 0);
}

