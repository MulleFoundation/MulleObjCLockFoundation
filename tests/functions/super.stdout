+[A foo]

+[B(C) foo]
+[A foo]
